habitat_baselines/rl/ppo/ppo_trainer.py

```diff
@@ -1014,12 +1014,12 @@ def _eval_checkpoint(
             observations, rewards_l, dones, infos = [
                 list(x) for x in zip(*outputs)
             ]
-            batch = batch_obs(
+            batch = batch_obs(  # type: ignore
```

Comment:
Why is this necessary? It wasn't before?

Reply:
mypy wasn't working correctly since habitat_baselines was missing a lot of `__init__.py` files.